### PR TITLE
Fix few issues for IBM cloud execution

### PR DESCRIPTION
**Pull request by petr-balogh**
Fixes: #2306

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -2507,3 +2512,15 @@ def measurement_dir(tmp_path):
         )
         os.mkdir(measurement_dir)
     return measurement_dir
+
+
+@pytest.fixture(scope="session", autouse=True)
+def ceph_toolbox(request):
+    """
+    This fixture initiates ceph toolbox pod for manually created deployment
```
Comment (vasukulkarni) [inline tests/conftest.py:2520]: I thought we had this already?

Comment (vasukulkarni) [reply tests/conftest.py:2520]: If it doesn't exist it does create it on the fly

File: ocs_ci/utility/localstorage.py [hunk 2]
```diff
@@ -1,20 +1,21 @@
...
+from ocs_ci.ocs import constants, defaults
 from ocs_ci.ocs.node import get_typed_nodes
-from ocs_ci.utility.retry import retry
 from ocs_ci.ocs.ocp import OCP
 from ocs_ci.ocs.resources import csv
 from ocs_ci.utility import utils
-from ocs_ci.ocs import constants, defaults
-from ocs_ci.framework import config
+from ocs_ci.utility.retry import retry
+from ocs_ci.utility.utils import clone_repo, run_cmd
```
Comment (clacroix12) [inline ocs_ci/utility/localstorage.py:18]: We are already importing the entire `utils` module on line 16. Can we not use `utils.clone_repo` and `utils.run_cmd` or change the `utils.<utility>` calls to be explicitly imported? We probably shouldn't be doing both.

File: tests/conftest.py (hunk 1 above)
Comment (petr-balogh) [reply tests/conftest.py:2520]: @vasukulkarni we didn't have it.  We do deploy cep_toolbox as part of our deployment but if the cluster was deployed manually and not via ocs-ci then no one deployed ceph toolbox pod and this fixture taking care of it on such clusters.

File: ocs_ci/utility/localstorage.py (hunk 2 above)
Comment (petr-balogh) [reply ocs_ci/utility/localstorage.py]: done

**Comment by petr-balogh** [comment]
After merge of some PR there was conflict in conftest.py as my new fixture is added at the end of file.

I consider this as verified as no code change done from last run.

**Pull request merged**